Dockerfile

```diff
@@ -22,6 +22,7 @@ COPY contracts /app/contracts
 COPY indexer /app/indexer
 COPY encoding /app/encoding
 COPY relay /app/relay
+COPY inabox /app/inabox
```

Comment:
Is this one needed?

Reply:
Yes. Docker Relay build fails without it.